files/en-us/web/css/css_anchor_positioning/using/index.md

```diff
@@ -453,27 +457,25 @@ selectElem.addEventListener("change", () => {
 });
 ```
 
-The result is as follows. Try selecting new `inset-area` values from the `<select>` menu to see the effect they have on the position of the infobox:
+Try selecting new `inset-area` values from the `<select>` menu to see the effect they have on the position of the infobox:
 
 {{ EmbedLiveSample("Using an `inset-area`", "100%", "250") }}
 
 ### Positioned element width
 
-In this example, we did not explicitly size the positioned element in either dimension. We deliberately omitted sizing to allow you to observe the behavior this causes.
+In the above example, we did not explicitly size the positioned element in either dimension. We deliberately omitted sizing to allow you to observe the behavior this causes.
 
-When a positioned element is placed into `inset-area` grid cells without explicit sizing, it aligns with the grid area specified and behaves as if {{cssxref("width")}} were set to {{cssxref("max-content")}}. It is sized according to its [containing block](/en-US/docs/Web/CSS/Containing_block) size, which is the width of its content. This size was imposed by setting `position: fixed`. Auto-sized absolutely and fixed-positioned elements are automatically sized so as to fit their contents, stretching as wide as needed to fit the text content, while constrained by the edge of the viewport. in this case, when placed on the left side of the grid with any `left` or `inline-start` value, the text wraps.
+When a positioned element is placed into `inset-area` grid cells without explicit sizing, it aligns with the grid area specified and behaves as if {{cssxref("width")}} were set to {{cssxref("max-content")}}. It is sized according to its [containing block](/en-US/docs/Web/CSS/Containing_block) size, which is the width of its content. This size was imposed by setting `position: fixed`. Auto-sized absolutely and fixed-positioned elements are automatically sized, stretching as wide as needed to fit the text content, while constrained by the edge of the viewport, but not growing to match the size of the anchor if `max-content` leads to a positioned element that is narrower or shorter than its anchor. In this case, when placed on the left side of the grid with any `left` or `inline-start` value, the text wraps.
```

Comment:
if the positioned element is just one character, it doesn't grow, so it may be narrower than the anchor